src/vulnpy/trigger/nosqli.py

```diff
@@ -0,0 +1,52 @@
+from pymongo import MongoClient
+import os
+
+
+def mongo_db_exists():
+    client = MongoClient(serverSelectionTimeoutMS=20)
+    db = client.pymongo_test
+
+    try:
+        db.posts.count_documents({})
+    except Exception as e:  # pragma: no cover
+        print("EXCEPTION", e)
+        return False
+
+    return True
+
+
+def create_mongo_client(**kwargs):
+    mongo_host = [os.environ.get("MONGODB_HOST", "localhost:27017")]
+    if mongo_db_exists():
+        return MongoClient(host=mongo_host, **kwargs)
+    else:
+        raise RuntimeError("Pymongo service is not running.")  # pragma: no cover
```

Comment:
I think it's important that vulnpy can still start up (with limited functionality) if we can't find the mongo server. Instead of a RuntimeError, can you print a warning?